### DISCUSSION TITLE
[Beta Projects Suggestion] Bulk Change

### DISCUSSION THREAD

**Discussion by airtonix** [community/community#29412] [Projects and Issues] [answered]
Frequently I want to make the same kind of changes to multiple project items(they aren't issue tickets yet): 

- assign to user
- convert to issue
- yadda yadda yadda.

Let us make selections composed of multiple items, then provide some kind of UI that lets us apply bulk changes: 

- a sidebar composed of all the projects fields as facets, dragging the multiple selection of items onto one of these applies the facet
- a button that launches a modal with all the fields as their respective ui control, changing one or more values here applies these changes to all selected items.

**Comment by rileybroughten** [comment, marked as answer]
Thank you for the feedback! Being able to bulk-apply changes to items is something we know users need! Marking this as a duplicate of #5465, keep an eye on that discussion post for updates! 👀
